ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml

```diff
@@ -173,6 +173,7 @@ tests:
     cluster_profile: gcp-qe
     test:
     - ref: openshift-extended-test
+    - ref: openshift-extended-web-tests
```

Comment:
it will change to chain. not sure how about your step. you could refer to openshift-e2e-test-qe.
maybe you need to make new chain.